jgiven-tests/src/test/java/com/tngtech/jgiven/report/html5/Html5AppStage.java

```diff
@@ -24,15 +25,19 @@
     protected File targetReportDir;
 
     protected void takeScreenshot() {
-        String base64 = ( (TakesScreenshot) webDriver ).getScreenshotAs( OutputType.BASE64 );
-        currentStep.addAttachment( Attachment.fromBase64( base64, MediaType.PNG ).withTitle( "Screenshot" ) );
+        String base64 = ((TakesScreenshot) webDriver).getScreenshotAs(OutputType.BASE64);
+        currentStep.addAttachment(Attachment.fromBase64(base64, MediaType.PNG).withTitle("Screenshot"));
     }
 
-    protected WebElement findTagWithName( String tagName ) {
-        List<WebElement> links = webDriver.findElements(
-            By.xpath( String.format( "//a/span[contains(@class,'tag') and contains(text(), '%s')]/..", tagName ) ) );
-        assertThat( links ).isNotEmpty();
-        return links.get( 0 );
+    protected WebElement findTagWithName(String tagName) {
+        By elementLocator = By.xpath(String.format("//a/span[contains(@class,'tag') and contains(text(), '%s')]/..",
```

Comment:
you could move this line down to the `findElements` command. They seem to belong together.